_posts/en/newsletters/2021-09-08-newsletter.md

```diff
@@ -0,0 +1,219 @@
+%}
+
+## Preparing for taproot #12: Vaults with taproot
+
+*A weekly [series][series preparing for taproot] about how developers
+and service providers can prepare for the upcoming activation of taproot
+at block height {{site.trb}}.*
+
+{% include specials/taproot/en/11-vaults-with-taproot.md %}
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [Bitcoin Core 22.0rc3][bitcoin core 22.0] is a release candidate
+  for the next major version of this full node implementation and its
+  associated wallet and other software. Major changes in this new
+  version include support for [I2P][topic anonymity networks] connections,
+  removal of support for [version 2 Tor][topic anonymity networks] connections,
+  and enhanced support for hardware wallets.
+
+- [Bitcoin Core 0.21.2rc2][bitcoin core 0.21.2] is a release candidate
+  for a maintenance version of Bitcoin Core.  It contains several bug
+  fixes and small improvements.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1
+repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][rust bitcoin repo], [BTCPay Server][btcpay server repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+- [Bitcoin Core #22009][] introduces a heuristic to quantify thriftiness
+  of [coin selection][topic coin selection] results. The heuristic can
+  be configured with the new option `-consolidatefeerate`, which
+  determines the boundary between consolidatory and thrifty behavior.
+  The wallet will now prefer the least wasteful input set from results
+  of the Branch and Bound (BnB) algorithm and the Knapsack algorithm.
+  Previously, the changeless BnB result was always preferred if one was
+  found. The subsequent PR #17526 aims to introduce a third coin
+  selection algorithm based on a single random draw from the wallet's
+  UTXO pool.
+
+- [Eclair #1907][] updates how it uses block chain watchdogs to prevent
+  [eclipse attacks][topic eclipse attacks] (see [Newsletter
+  #123][news123 eclair watchdogs]).  When Tor is available, Eclair now
+  uses it to contact the watchdogs (using native onion endpoints when
+  possible).  This should make it more difficult for an upstream
+  attacker to selectively censor just the watchdog providers.
+
+- [Eclair #1910][] updates how it uses Bitcoin Core's ZMQ messaging
+  interface to improve the reliability of learning about new blocks.
+  Anyone else using ZMQ for block discovery may wish to investigate the
+  changes.
+
+- [BIPs #1143][] introduces BIPs 380-386 specifying
+  [output script descriptors](). Output script descriptors are a simple
```

Comment:
Wasn't sure if a rehash of what output script descriptors was helpful at all so feel free to remove it.